### DISCUSSION TITLE
Access Private key

### DISCUSSION THREAD

**Discussion by chamid** [trustwallet/wallet-core#2465] [Q&A]
Is there a way to access plain text of private key???
I'm want to sign a tron transaction and i need to know that is my Private Key in correct format or not

**Comment by hewigovens** [comment]
Can you elaborate your issue? access private key from what, keystore json or `HDWallet`?

**Reply by ffoDevilSusiJ** [reply]
> Can you elaborate your issue? access private key from what, keystore json or `HDWallet`?

What is the keystore?
